src/index.css

```diff
@@ -16,3 +16,28 @@ code {
 h1 {
     text-align: center;
 }
+
+.red-button{
+    background-color: red !important;
+    color: white !important;
+    border-color: transparent !important;
+}
+
+.blue-button{
+    background-color: #216894 !important;
+    color: white !important;
+    border-color: transparent !important;
+}
+
+.red-button:hover{
+    background-color: white !important;
+    color: red !important;
+    border-color: red !important;
+}
+
+.blue-button:hover{
+    background-color: white !important;
+    color: #216894 !important;
+    border-color: #216894 !important;
+}
+
```

Comment:
I don't like all this important!. You can use the theme provider from page template I believe. Something like this? Or maybe you already tried it. https://stackoverflow.com/questions/46486565/mui-customize-button-color

Reply:
We tried to but without success, as mui has more specificity than us it smash our changes. and inline styling generate syntax errors